magnum_capi_helm/driver.py

```diff
@@ -610,6 +688,12 @@ def _update_helm_release(self, context, cluster, nodegroups=None):
                 if ng.role != NODE_GROUP_ROLE_CONTROLLER
             ],
             "addons": {
+                "openstack": {
+                    "csiCinder": {
+                        "defaultStorageClass": default_storageclass,
+                        "additionalStorageClasses": add_storageclasses,
```

Comment:
Instead of returning a tuple, why not have `_storageclass_definitions` return this object? Seems cleaner.

Reply:
Yes that makes sense. Changed.